specification.markdown

```diff
@@ -535,6 +535,40 @@ An `<element>` SHALL NOT have more than one `<implementation_note>` of the same
 
 The note\_attribute attribute is REQUIRED.
 
+#### \<implementation_note> Example
+
+The following fragment of an EBML Schema demonstrates how an `<implementation_note>` is used. In this case an EBML Schema documents a list of items that are described with an optional cost. The Currency Element uses an `<implementation_note>` to say that the Currency Element is REQUIRED if the Cost Element is set, otherwise not.
+
+```xml
+<element name="Item" path="1*1(\Items\)" id="0x4025" type="master">
+  <documentation lang="en" purpose="definition">
+    An set of items.
+  </documentation>
+</element>
+<element name="Item" path="0*(\Items\Item)" id="0x4026"
+  type="master">
+  <documentation lang="en" purpose="definition">
+    An item.
+  </documentation>
+</element>
+<element name="Cost" path="0*1(\Items\Item\Cost)" id="0x4024"
+  type="float" minOccurs="0" maxOccurs="1">
+  <documentation lang="en" purpose="definition">
+    The cost of the item, if any.
+  </documentation>
+</element>
+<element name="Currency" path="0*1(\Items\Item\Currency)" id="0x403F"
```

Comment:
`maxOccurs` 1 is not the default value so should be set in Schema.

Reply:
There's a question in #292 whether the minOccurs should be included in the Schema since the value may change depending on the context. Here we see there's another issue, the minOccurs is also included in the path. Here it's the default value, so not writing it also has a different meaning (it's the default).

Reply:
Resolved as it matches the current expected behavior. It may have to be changed once we find a solution for #292